examples/cloudflare-redwood/alchemy.run.ts

```diff
@@ -0,0 +1,24 @@
+import "alchemy/cloudflare";
+
+import alchemy from "../../alchemy/src";
+import { D1Database, Redwood } from "../../alchemy/src/cloudflare";
+
+const app = await alchemy("redwood-app");
+
+const database = await D1Database("redwood-db", {
+  name: "redwood-db",
+  migrationsDir: "drizzle",
+});
```

Comment:
I wonder if this should be made a default in `Redwood` since it's needed, right? @lockr7